### DISCUSSION TITLE
Feature request: Themes for syntax highlighting

### DISCUSSION THREAD

**Discussion by james-pre** [community/community#122288] [Other Feature Feedback, Questions, & Ideas]
Whenever I'm looking at code on Github, I always feel uncomfortable because of the theme colors. I've gotten used to seeing various symbols in a specific color, so when I look at code on Github, it takes a lot more effort to read. Customizable code themes on Github would provide a huge productivity boost since reading and understanding code will be faster due to the familiarity with the colors involved.

**Comment by Panonim** [comment]
I don't think it is a place for that, also github 90% wont realse new theme.
